ui/app/AppLayouts/Chat/views/communities/CommunityPermissionsView.qml

```diff
@@ -0,0 +1,65 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+import SortFilterProxyModel 0.2
+import utils 1.0
+
+import "../../../Chat/controls/community"
```

Comment:
please use qualified imports whenever possible. In this case `import AppLayouts.Chat.controls.community 1.0` but probably qmldir needs to be updated. Here and in some other places.

Reply:
Done! Updated all imports related to the task.